docs/dev/awips-development-environment.md

```diff
@@ -41,11 +43,15 @@ proxy=_none_
 
 ## 3. Install the ADE
 
+!!! note "The following may need to be run as *root* user."
```

Comment:
For some reason this note isn't showing up with the correct box (but the one above is...)

![image](https://user-images.githubusercontent.com/65614396/119003164-20e16a80-b953-11eb-9b08-4058d1fb36b2.png)